### ISSUE TITLE
Make URLs in multiple lines completly clickable in article view

### ISSUE THREAD

**Issue by Eucyon** [newsboat/newsboat#564]
I am using LXTerminal.

When I cat a file containing an URL, LXTerminal underlines the complete URL even if it stands in multiple lines and makes it clickable.

URLs in more than one line shown in the article view are underlined only in the upper line and therefor uncomplete. If I copy them and paste, they contain a line break.

**Comment by Minoru** [comment]
This is a known limitation in ncurses. I described it in more detail here: https://github.com/akrennmair/newsbeuter/issues/282#issuecomment-222318425 The fix for this depends on #232; I'm closing this in favour of that issue.

**Comment by dennisschagt** [comment]
After replacing STFL, we might be able to use the [`OSC 8` escape sequence](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda) to make the full URL available even if only a part of it is visible.

I don't think it is possible to use it with ncurses and definitely isn't supported while we are still using STFL.
